build.gradle

```diff
@@ -18,128 +18,171 @@ plugins {
-    // Declares the type and metadata of the CPK (this CPB has one CPK).
-    workflow {
-        name "ModuleNameHere"
-        versionId 1
-        vendor "VendorNameHere"
-    }
-}
+// subprojects {
+allprojects {
+    group 'com.r3.hellocorda'
+    version '1.0-SNAPSHOT'
+
+    def javaVersion = VERSION_11
+
 
 // Declare the set of Kotlin compiler options we need to build a CorDapp.
-tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).configureEach {
-    kotlinOptions {
-        allWarningsAsErrors = false
-
-        // Specify the version of Kotlin that we are that we will be developing. 
-        languageVersion = '1.7'
-        // Specify the Kotlin libraries that code is compatible with
-        apiVersion = '1.7'
-        // Note that we Need to use a version of Kotlin that will be compatible with the Corda API.
-        // Currently that is developed in Kotlin 1.7 so picking the same version ensures compatibility with that.
-
-        // Specify the version of Java to target.
-        jvmTarget = javaVersion
-
-        // Needed for reflection to work correctly.
-        javaParameters = true
-
-        // -Xjvm-default determines how Kotlin supports default methods.
-        // JetBrains currently recommends developers use -Xjvm-default=all
-        // https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.jvm/-jvm-default/
-        freeCompilerArgs += [
-                "-Xjvm-default=all"
-        ]
+    tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).configureEach {
+        kotlinOptions {
+            allWarningsAsErrors = false
+
+            // Specify the version of Kotlin that we are that we will be developing.
+            languageVersion = '1.7'
+            // Specify the Kotlin libraries that code is compatible with
+            apiVersion = '1.7'
+            // Note that we Need to use a version of Kotlin that will be compatible with the Corda API.
+            // Currently that is developed in Kotlin 1.7 so picking the same version ensures compatibility with that.
+
+            // Specify the version of Java to target.
+            jvmTarget = javaVersion
+
+            // Needed for reflection to work correctly.
+            javaParameters = true
+
+            // -Xjvm-default determines how Kotlin supports default methods.
+            // JetBrains currently recommends developers use -Xjvm-default=all
+            // https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.jvm/-jvm-default/
+            freeCompilerArgs += [
+                    "-Xjvm-default=all"
+            ]
+        }
     }
-}
 
-repositories {
-    // All dependencies are held in Maven Central
-    mavenCentral()
+    repositories {
+        // All dependencies are held in Maven Central
+        mavenCentral()
+        maven {
+            url = "$artifactoryContextUrl/"
+        }
+
+
+        /*
+    // Internal artifactory repos for dev

```

Comment:
I know why you've commented these out ( to test the RC) , but i'd question if its correct to have these committed to release branch like this.  Having large amounts of commented out code doesn't look good in our default branch of a public repo IMO.

Reply:
agreed